reference/markdown/Format-comparison.md

```diff
@@ -0,0 +1,34 @@
+Although the original [Markdown DITA](./Markdown-syntax-reference.md) format and the [MDITA](./MDITA-syntax-reference.md) format for _LwDITA_ share some common syntax, there are several differences to consider when choosing which format to use.
+
+<!-- Re-use short descriptions from syntax topics -->
+
+- In 2015, the original _DITA-OT Markdown_ plug-in introduced a series of conventions to convert [Markdown] content to DITA, and vice-versa. This Markdown flavor was called _“Markdown DITA”_. The `markdown` format adds several complementary constructs to represent DITA content in Markdown, beyond those proposed for the [MDITA](./MDITA-syntax-reference.md) format in the [Lightweight DITA][LwDITA] specification drafts.
+
+- In 2017, the Markdown plug-in was superseded by the _LwDITA_ plug-in, which was bundled with DITA-OT 3.0, and added new formats for [Lightweight DITA][LwDITA]. The `mdita` format implements the subset of Markdown features proposed in the latest specification drafts, but differs in some ways from the original [Markdown DITA](./Markdown-syntax-reference.md) format.
+
+The following table provides an overview of differences between the `markdown` and `mdita` formats.
+
+| Features                 | Markdown DITA                   | MDITA (LwDITA)             |
+| ------------------------ | ------------------------------- | -------------------------- |
+| Format                   | `markdown`                      | `mdita`                    |
```

Comment:
This is `@format` attribute in e.g. DITA Maps, but it's also the file extension. So should this be `Format / File extension`?